internal/github/contributions.go

```diff
@@ -77,6 +125,33 @@ func (g *GithubService) GetContributionsByUsername(ctx context.Context, username
 	return contributions, nil
 }
 
+func (g *GithubService) GetFirstContributionYearByUsername(ctx context.Context, username string) (*time.Time, error) {
```

Comment:
what happens if someone hasn't contributed?

Reply:
ah just tested this by creating a new user and it appears creating a user gives you one contribution lol

https://github.com/nocontributions